File: data/imported/knowledge/slides/net/merged/snippets/snippet_029.cs
[Fact]
    public void TestSoftEdge()
    {
        // Soft edge radius persists.
        using var pres = new Presentation();
        var slide = pres.Slides[0];
        slide.Shapes!.Clear();
        var shape = slide.Shapes.AddAutoShape(ShapeType.Rectangle, 100, 100, 200, 100);
        var ef = shape.EffectFormat;
        ef.EnableSoftEdgeEffect();
        ef.SoftEdgeEffect!.Radius = 10;

        using var pres2 = TestHelpers.SaveAndReopen(pres, _tempDir);
        var se2 = pres2.Slides[0].Shapes![0].EffectFormat.SoftEdgeEffect;
        se2.Should().NotBeNull("soft_edge_effect should not be None after reload");
        se2!.Radius.Should().Be(10);
    }

File: data/imported/knowledge/slides/java/merged/snippets/snippet_029.java
@Test
    void testGlow() throws IOException {
        try (var pres = new Presentation()) {
            var slide = clear(pres);
            var shape = slide.getShapes().addAutoShape(ShapeType.ELLIPSE, 100, 100, 200, 200);
            var ef = shape.getEffectFormat();
            ef.enableGlowEffect();
            ef.getGlowEffect().setRadius(15);
            ef.getGlowEffect().getColor().setColor(Color.GOLD);

            try (var pres2 = saveAndReopen(pres)) {
                var g2 = pres2.getSlides().get(0).getShapes().get(0).getEffectFormat().getGlowEffect();
                assertThat(g2).as("glow_effect should not be None after reload").isNotNull();
                assertThat(g2.getRadius()).isEqualTo(15);
            }
        }
    }